wordpress-mobile/WordPress-iOS
# [Activity Log] Add support for free WPCom sites.

This adds support for free WPCom sites to AL.

I'll submit a separate PR to update the Podfile after the changed to WPKit are merged (https://github.com/wordpress-mobile/WordPressKit-iOS/pull/26) and a release is tagged.

To test:
1. Manually edit Podfile so it points to `4d1d004` commit in `WordPressKit-iOS`
1. Go to AL with a free site that has some activity
1. Verify that activity shows up
1. Go to AL with a free site that doesn't have any activity yet (just create one if you don't have one handy)
1. Verify that the empty state shows up
